src/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cs

```diff
@@ -511,7 +529,7 @@ public static void DotNetCoreTest(this ICakeContext context, string project)
         ///         Configuration = "Release"
         ///     };
         ///
-        ///     DotNetCoreTest("./test/Project.Tests", settings);
+        ///     DotNetCoreTest("./test/Project.Tests/Project.Tests.csproj", settings);
```

Comment:
@henningst can we get the same thing on this alias?  They appear on different pages on the website, so the information shoudl appear on both.  Thanks!